src/model/delta/transform.js

```diff
@@ -350,10 +354,21 @@ function padWithNoOps( deltas, howMany ) {
 // Using data given in `context` object, sets `context.insertBefore` and `context.forceNotSticky` flags.
 // Also updates `context.wasAffected`.
 function _setContext( a, b, context ) {
+	_setBWasUndone( b, context );
```

Comment:
Your question is almost correct ;).

We need only information about delta `b`, because delta `a` cannot be undone delta, because we are transforming it at the moment, so it is not yet applied. For delta `a` it would always be `false`.